code/src/components/TotalTodos.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const TotalTodos = ({ tasks, completedTodos }) => {
+  return (
+    <div>
+      Completed:
+      {completedTodos.length}/{tasks.length}
+    </div>
+  );
+};
+
+export default TotalTodos;
```

Comment:
Is this component being used somewhere? couldn't see it.. 
Maybe if it's not being used it can be deleted?